lib/vat_info/models/vat_payer.rb

```diff
@@ -0,0 +1,49 @@
+module VatInfo
+  module Models
+    class VatPayer
+      attr_accessor :data
+
+      def initialize(params)
+        @data                       = {}
```

Comment:
pokud voláš params.fetch(neco, nil) tak je to stejný jako params[neco], takže tu můžeš ty fetche klidně odstranit a nahradit jen [].

Reply:
fixed